providers/summary/clearlydefined.js

```diff
@@ -91,7 +91,7 @@ class ClearlyDescribedSummarizer {
         'licensed.declared',
         license
           .split('/')
-          .map(normalizeSpdx)
+          .map(correct)
```

Comment:
are you going to pass in the option to turn off upgrades?

I noticed this being called in multiple places. If we care about the params being passed in, should we have a helper method that everyone uses to ensure consistency? Perhaps revive `normalizeSPDX` but have it just do `correct` with the designated args.